addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -930,8 +930,6 @@ export class PosStore extends Reactive {
             [this.pos_session.id],
         ]);
         const ordersDetails = closingData.orders_details;
-        const paymentsAmount = closingData.payments_amount;
-        const payLaterAmount = closingData.pay_later_amount;
```

Comment:
since you are removing these here because they are unused, you may remove them as well from the return value of "get_closing_control_data".